README.rst

```diff
@@ -166,6 +172,13 @@ Goals for the project
 - We would love to see the Airflow community members create, maintain and share their providers to build an Ecosystem
   of Providers.
 
+Limitations
+-----------
+
+- In Airflow sensors have a param ``mode`` which has two types poke and reschdule,
+  which uses the worker slot to poke or at intervals respectively. Whereas in async sensors this param has
```

Comment:
This sentence needs to be grammatically corrected as mentioned above, and avoid writing big sentences please

Reply:
Changed it.